server/internal/core/ports/tx_decoder.go

```diff
@@ -0,0 +1,14 @@
+package ports
+
+import "github.com/ark-network/ark/server/internal/core/domain"
+
+type TxIn domain.VtxoKey
+
+type TxOut struct {
+	Amount   uint64
+	PkScript []byte
+}
```

Comment:
```suggestion
type TxOut wire.TxOut
```

Reply:
ignored as we loop over these outs to create `domain.Vtxo`, we never translate them to `wire.TxOut`